model.go

```diff
@@ -154,6 +154,15 @@ type Model interface {
 	PasswordHash() string
 
 	AddBlockDevice(string, BlockDeviceArgs) error
+
+	// SetEnvironVersion sets the environment version to the specified value.
+	//
+	// The version must be a non-negative integer. The caller is responsible
```

Comment:
Thanks @CodingCookieRookie, I would avoid making this assertion as this is business logic that belongs in the Juju controller and not in the controller. I don't believe we do any validation at the moment because it is a static controlled value in the code.

I think just add a line saying "environ version represents the version of the environ provider that the Juju model is currently using."

Reply:
Yup seems to be hardcoded, sure thing thanks tom :)